processor/src/test/java/org/mapstruct/ap/test/conversion/bignumbers/BigNumbersConversionTest.java

```diff
@@ -188,4 +190,22 @@ public void shouldApplyReverseBigDecimalConversions() {
         assertThat( source.getString() ).isEqualTo( new BigDecimal( "13.45" ) );
         assertThat( source.getBigInteger() ).isEqualTo( new BigDecimal( "14" ) );
     }
+
+    @Test
+    @IssueKey("1009")
+    @WithClasses({ BigIntegerSource.class, BigIntegerTarget.class, BigIntegerMapper.class })
+    public void shouldNotGenerateCreateDecimalFormatMethod() {
+
+        Class clz = BigIntegerMapper.INSTANCE.getClass();
+        try {
+            Method method = clz.getDeclaredMethod( "createDecimalFormat", String.class );
+            assertThat( method ).isNull();
+        }
+        catch ( NoSuchMethodException ex ) {
```

Comment:
A bit nitpicking :smile:, but you can use our `GeneratedSource` JUnit Rule for this.  See the usages of it to see how you can do it